include/pingcap/kv/RegionCache.h

```diff
@@ -159,15 +156,13 @@ struct RPCContext
     metapb::Peer peer;
     Store store;
     std::string addr;
-    std::vector<uint64_t> all_stores;
```

Comment:
`all_stores` will be setup in `buildBatchCopTask()` by calling `getAllValidTiFlashStores()`